packages/storage/src/metadata/gundbMetadataStore.ts

```diff
@@ -182,6 +183,26 @@ export class GundbMetadataStore implements UserMetadataStore {
     return schema;
   }
 
+  /**
+   * {@inheritDoc @spacehq/sdk#UserMetadataStore.setNotificationsLastSeenAt}
+   */
+  public async setNotificationsLastSeenAt(timestamp: number): Promise<void> {
+    const encryptedTimestamp = await this.encrypt(timestamp.toString());
+    const lookupKey = this.getNotificationsLastSeenAtLookupKey();
+    const nodeRef = this.lookupUser.get(lookupKey).put({ data: encryptedTimestamp });
+    this.listUser.get(NotificationsLastSeenAtCollection).set(nodeRef as unknown as EncryptedMetadata);
```

Comment:
I don't think you need to add it to a `set()` here. The set is useful if we need a list of all last seen at timestamp right? And I don't think we would need that. We are only interested in setting and getting that data right? So the `put()` should be fine. It would always override the latest value.